metaflow/metaflow_config.py

```diff
@@ -154,6 +154,7 @@ def from_conf(name, default=None):
 ARGO_RUN_URL_PREFIX = from_conf("ARGO_RUN_URL_PREFIX", "")
 METAFLOW_RUN_URL_PREFIX = from_conf("METAFLOW_RUN_URL_PREFIX", "")
 KFP_MAX_PARALLELISM = int(from_conf("KFP_MAX_PARALLELISM", 10))
+KFP_MAX_CONCURRENCY = int(from_conf("KFP_MAX_CONCURRENCY", 10))
```

Comment:
Since this is a new parameter, do we want to make it not KFP specific and more explicitly related to the cron workflow? Maybe `CRON_MAX_CONCURRENCY`?

Reply:
These KFP_ prefixes are a way to namespace Metaflow config by the `KFP_` plugin, so KFP_MAX_CONCURRENCY is a sister to KFP_MAX_PARALLELISM